arm-powerbiembedded/2016-01-29/swagger/powerbiembedded.json

```diff
@@ -0,0 +1,1041 @@
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Workspace collection created successfully",
+                        "schema": {
+                            "$ref": "#/definitions/WorkspaceCollection"
+                        }
+                    },
+                    "default": {
+                        "description": "Default Response. It will be deserialized as per the Error defintion specified in the schema. Exception will be thrown.",
+                        "schema": {
+                            "$ref": "#/definitions/Error"
+                        }
+                    }
+                }
+            },
+            "put": {
+                "tags": [
+                    "WorkspaceCollections"
+                ],
+                "operationId": "WorkspaceCollection_create",
+                "description": "The **Create Power BI Workspace Collection** operation creates a new Power BI Workspace Collection with the specified properties. A Power BI Workspace Collection contains one or more Power BI Workspaces and can be used to provision keys that provide API access to those Power BI Workspaces.",
+                "consumes": [
+                    "application/json"
+                ],
+                "produces": [
+                    "application/json"
+                ],
+                "parameters": [
+                    {
+                        "name": "subscriptionId",
+                        "in": "path",
+                        "description": "Azure subscription id",
+                        "required": true,
+                        "type": "string"
+                    },
+                    {
+                        "name": "resourceGroupName",
+                        "in": "path",
+                        "description": "Azure resource group",
+                        "required": true,
+                        "type": "string"
+                    },
+                    {
+                        "name": "workspaceCollectionName",
+                        "in": "path",
+                        "description": "Power BI Embedded workspace collection name",
+                        "required": true,
+                        "type": "string"
+                    },
+                    {
+                        "name": "api-version",
+                        "in": "query",
+                        "description": "Azure API version",
+                        "required": true,
+                        "type": "string"
+                    },
+                    {
+                        "name": "body",
+                        "in": "body",
+                        "description": "Create workspace collection request",
+                        "required": true,
+                        "schema": {
+                            "$ref": "#/definitions/CreateWorkspaceCollectionRequest"
```

Comment:
The current model name CreateWorkspaceCollectionRequest is fine. However, CreateWorkspaceCollectionParameters would be more in sync with other specs.